src/utils.ts

```diff
@@ -19,3 +21,64 @@ export function numberWithCommas(number: number) {
 }
 
 export const calcRate = (num1: number, num2: number) => (num1 / num2) * 100;
+
+export function useScroll() {
+  const [isScrolled, setIsScrolled] = useState(false);
+
+  useEffect(() => {
+    const handleScroll = _.debounce(() => {
+      setIsScrolled(!!window.scrollY);
+    }, 100);
+
+    window.addEventListener('scroll', handleScroll);
+
+    return () => window.removeEventListener('scroll', handleScroll);
+  });
+  console.log(isScrolled);
+
+  return isScrolled;
+}
+
+export function usePageInfo() {
+  const pages = [
+    {
+      path: '/create',
+      title: '여행 추가'
+    },
+    {
+      path: '/trip/[0-9]+/expense',
+      title: '쓴 돈 추가'
+    }
+  ];
+
+  const currentUrl = window.location.href;
+  const result = pages.find((p) => currentUrl.match(p.path));
+
+  if (result === undefined) {
+    return;
+  }
+  return result.title;
+}
+
+export function typeToEmoji(type: string) {
```

Comment:
Animal Enum을 동물 이모지 string으로 바꿔주는 함수입니다! 
```suggestion
export function typeToEmoji(type: Animals) {
``` 
지금 보니 이렇게 차후 수정하는게 낫겠네용..